<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>200</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>400</width>
    <height>200</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>400</width>
    <height>200</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>CurrencyConverter - pythonexplainedto.me</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="verticalLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>10</y>
      <width>221</width>
      <height>52</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <property name="spacing">
      <number>4</number>
     </property>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <property name="spacing">
        <number>10</number>
       </property>
       <item>
        <widget class="QLabel" name="amountLabel">
         <property name="font">
          <font>
           <family>Calibri</family>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Amount:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="amountLine">
         <property name="font">
          <font>
           <family>Calibri</family>
           <pointsize>12</pointsize>
          </font>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QLabel" name="resultLabel">
       <property name="font">
        <font>
         <family>Calibri</family>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>70</y>
      <width>381</width>
      <height>104</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <property name="spacing">
      <number>5</number>
     </property>
     <property name="leftMargin">
      <number>0</number>
     </property>
     <property name="topMargin">
      <number>5</number>
     </property>
     <property name="rightMargin">
      <number>0</number>
     </property>
     <property name="bottomMargin">
      <number>5</number>
     </property>
     <item>
      <widget class="Line" name="line">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QGridLayout" name="gridLayout">
       <property name="leftMargin">
        <number>1</number>
       </property>
       <property name="rightMargin">
        <number>1</number>
       </property>
       <property name="spacing">
        <number>3</number>
       </property>
       <item row="0" column="0">
        <widget class="QLabel" name="label">
         <property name="font">
          <font>
           <family>Calibri</family>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="text">
          <string>From:</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QLabel" name="label_2">
         <property name="font">
          <font>
           <family>Calibri</family>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="text">
          <string>To:</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QComboBox" name="fromCurrency">
         <property name="font">
          <font>
           <family>Calibri</family>
           <pointsize>12</pointsize>
          </font>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QComboBox" name="toCurrency">
         <property name="font">
          <font>
           <family>Calibri</family>
           <pointsize>12</pointsize>
          </font>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QPushButton" name="btnConvertCurrency">
       <property name="font">
        <font>
         <family>Calibri</family>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="text">
        <string>ConvertCurrency</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </widget>
   <widget class="QLabel" name="appbg">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>400</width>
      <height>200</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <zorder>appbg</zorder>
   <zorder>verticalLayoutWidget_2</zorder>
   <zorder>verticalLayoutWidget</zorder>
  </widget>
  <widget class="QStatusBar" name="statusbar">
   <property name="sizeGripEnabled">
    <bool>false</bool>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
